apps/rehabstod/src/components/Form/SelectMultiple.tsx

```diff
@@ -1,31 +1,104 @@
-import { IDSCheckboxGroup, IDSSelectMultiple } from '@frontend/ids-react-ts'
-import { ReactNode } from 'react'
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  autoUpdate,
+  flip,
+  FloatingFocusManager,
+  FloatingPortal,
+  offset,
+  size,
+  useDismiss,
+  useFloating,
+  useInteractions,
+  useRole,
+} from '@floating-ui/react'
+import { IDSCheckboxGroup, IDSIcon } from '@frontend/ids-react-ts'
+import { ReactNode, useId, useState } from 'react'
+import { classNames } from '../../utils/classNames'
 import { TooltipIcon } from '../TooltipIcon/TooltipIcon'
+import { Input } from './Input'
 
 export function SelectMultiple({
   label,
   description,
   options,
-  id,
   placeholder,
 }: {
   label: string
   description: string
   options: ReactNode
-  id: string
   placeholder: string
 }) {
+  const [open, setOpen] = useState(false)
+  const { x, y, strategy, refs, context } = useFloating({
+    placement: 'bottom-start',
+    open,
+    onOpenChange: setOpen,
+    whileElementsMounted: autoUpdate,
+    middleware: [
+      offset(5),
+      flip({ padding: 10 }),
+      size({
+        apply({ rects, elements, availableHeight }) {
+          Object.assign(elements.floating.style, {
+            maxHeight: `${Math.min(availableHeight, 400)}px`,
+            width: `${rects.reference.width}px`,
+          })
+        },
+        padding: 10,
+      }),
+    ],
+  })
+  const dismiss = useDismiss(context)
+  const role = useRole(context, { role: 'listbox' })
+  const { getFloatingProps } = useInteractions([dismiss, role])
+  const id = useId()
+
   if (!options || (options instanceof Array && options.length === 0)) {
     return null
   }
 
   return (
-    <>
+    <div>
       <label htmlFor={id}>{label}</label>
       {description && <TooltipIcon description={description} name="question" size="s" className="relative top-1 ml-2" />}
-      <IDSSelectMultiple placeholder={placeholder} className="text-sm">
```

Comment:
We set the text to sm during a UX-meeting so that the lists wouldn't be to large.

Reply:
This makes it different size from the NumberInput, looks really strange imo. I'll talk to UX and get a second opinion.

Reply:
Otherwise maybe we can introduce a scroll later

Reply:
Talked to UX, it should be 16 for input fields, but it's ok for table cells to have 14px font size. Perhaps there was some confusion here. But we have the go-ahead for removing "text-sm" 😃